user-guide/grpc.md

```diff
@@ -199,7 +199,7 @@ spec:
   hosts:
   - "*"
   secret: ambassador-cert
-  alpn_protocol: h2
+  alpn_protocols: h2
```

Comment:
It disturbs me that I had to go look this up. :wink: (the plural is indeed correct)